coderedcms/templates/coderedcms/formfields/mailchimp/subscriber_integration_js.html

```diff
@@ -0,0 +1,127 @@
+<script>
```

Comment:
Ehh.. this js is a bit on the nasty side... It makes sense due to the dynamic nature of the integration, but we really need to find a better way to handle this kind of thing in the future. It's fine for now, but I will create an issue for all of the instances where we have to run JS through the django template engine. That can be an issue for Will maybe during pycon sprints.